### ISSUE TITLE
"pod install" fails with "Xcodeproj relies on a library called `fiddle` to read and write Xcode project files"

### ISSUE THREAD

**Issue by OliverPearmain** [CocoaPods/Xcodeproj#295]
I've just downloaded an iOS/objective-c project from github which uses Cocoapods for its dependencies. 

When I attempt to `pod install` in the terminal I receive the following error: 
```shell
    Updating local specs repositories
    Analyzing dependencies
    [!] Xcodeproj relies on a library called `fiddle` to read and write Xcode project files. Ensure your Ruby installation includes `fiddle` and try again.
```

So I installed fiddle using `gem install fiddle` which succeeded successfully and if I now perform a `gem list` it would appear that `fiddle (0.6.3)` is successfully installed.  Addtionally I can see a `/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/fiddle-0.6.3/` directory.

However when I try to `pod install` I get the exact same error as before.

If it makes any difference, I am using RBENV. 

Can anyone help?  I've googled the error to try and resolve myself but the only results I get are the source code for this project that outputs the error itself.  I've also posted on StackOverflow but thats so far gone unanswered. 

http://stackoverflow.com/questions/31855134/cocoapods-pod-install-fails-with-xcodeproj-relies-on-a-library-called-fiddle

Some further info incase it helps: 
```shell
which ruby
/Users/myname/.rbenv/shims/ruby
```
```shell
pod install --verbose
  Preparing

Updating local specs repositories

Updating spec repo `master`
  $ /usr/bin/git pull --ff-only
  From https://github.com/CocoaPods/Specs
	 194b47f..be34c13  master     -> origin/master
  Updating 194b47f..be34c13
  Fast-forward
   .../0.1.2/Backgroundable.podspec.json              | 21 +++++++++++++++++
   .../1.0.0/ToolTipNavigation.podspec.json           | 27 ++++++++++++++++++++++
   2 files changed, 48 insertions(+)
   create mode 100644 Specs/Backgroundable/0.1.2/Backgroundable.podspec.json
   create mode 100644 Specs/ToolTipNavigation/1.0.0/ToolTipNavigation.podspec.json

Analyzing dependencies

Inspecting targets to integrate
[!] Xcodeproj relies on a library called `fiddle` to read and write Xcode project files. Ensure your Ruby installation includes `fiddle` and try again.


/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-0.26.3/lib/xcodeproj/plist_helper.rb:7:in `rescue in <top (required)>'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-0.26.3/lib/xcodeproj/plist_helper.rb:1:in `<top (required)>'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:69:in `require'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:69:in `require'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-0.26.3/lib/xcodeproj/project.rb:7:in `<top (required)>'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer/analyzer/target_inspector.rb:32:in `compute_results'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer/analyzer.rb:655:in `block (2 levels) in inspect_targets_to_integrate'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer/analyzer.rb:653:in `each'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer/analyzer.rb:653:in `block in inspect_targets_to_integrate'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/user_interface.rb:59:in `section'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer/analyzer.rb:652:in `inspect_targets_to_integrate'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer/analyzer.rb:61:in `analyze'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:210:in `analyze'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:133:in `block in resolve_dependencies'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/user_interface.rb:59:in `section'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:132:in `resolve_dependencies'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/installer.rb:104:in `install!'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/command/project.rb:101:in `run'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/lib/cocoapods/command.rb:48:in `run'
/Users/myname/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/cocoapods-0.38.2/bin/pod:44:in `<top (required)>'
/Users/myname/.rbenv/versions/2.2.0/bin/pod:23:in `load'
/Users/myname/.rbenv/versions/2.2.0/bin/pod:23:in `<main>'
```

**Comment by segiddins** [comment]
Sounds like an issue with your Ruby install -- fiddle should come built in by default on everything 2.0 and later.

**Comment by OliverPearmain** [comment]
Firstly thank you for taking the time to read about my issue and reply.  I'm not very confident when it comes to ruby/shell/bash/package installers/command line shenanigans since historically every time I touch this stuff something goes wrong.  I did't want to bugger up other stuff I have dependant on my existing ruby install so I thought I'd ask for help first.  I have heeded your advice and installed a newer version of ruby and all is now well in the world, so thanks for encouraging me to take that step, you made my day.

In case it helps anyone here are the commands I executed...

```shell
brew update
brew upgrade rbenv ruby-build
rbenv install 2.2.2
rbenv global 2.2.2
rbenv rehash
gem install cocoapods
pod install
```
